ui/admin/app/components/form/field/list-wrapper/key-value/index.hbs

```diff
@@ -35,14 +35,14 @@
                   'form/field/list-wrapper/key-value/text'
                   disabled=@disabled
                   value=option.key
-                  ariaLabel=keyHeaderID
+                  aria-labelledby=keyHeaderID
```

Comment:
if we can't pass it down by `...attributes`, I would keep the standard camelcase and call it `ariaLabeledBy`

Reply:
Oh yes. Good suggestion. I will fix that now.